### DISCUSSION TITLE
Having an issue constraining a div

### DISCUSSION THREAD

**Discussion by astarrh** [tailwindlabs/tailwindcss#12535] [Help] [answered]
Hello, 

I am having a little trouble preventing a div set to `h-full` from expanding with it's content. I want the column to fill the space if it is empty, but it needs to stay within it's boundaries and allow an inner div to be in charge of handling overflow with scroll.

I have been trying to get it to work in the leftmost column in this example (content shouldn't be allowed below the red line):

https://play.tailwindcss.com/dXYgxmJUcd

**Comment by wongjn** [comment, marked as answer]
Element children of vertical flex layouts have `min-height: min-content` applied to them implicitly. We'd need to override this with a smaller value (say with `min-height: 0`) to allow the content to shrink appropriately: https://play.tailwindcss.com/7b6VrG2Qf2

**Reply by astarrh** [reply]
Perfect! Thanks much
